protocol/build.sbt

```diff
@@ -19,8 +19,8 @@
 // JSON DEPENDENCIES
 //
 libraryDependencies ++= Seq(
-  "com.typesafe.play" %% "play-json" % "2.3.10" excludeAll( // Apache v2
-      ExclusionRule(organization = "com.fasterxml.jackson.core")
+  "com.typesafe.play" %% "play-json" % "2.5.9" excludeAll( // Apache v2
+      //ExclusionRule(organization = "com.fasterxml.jackson.core")
```

Comment:
Is there a reason you got rid of this exclusion rule? Isn't jackson coming from Apache Spark? I would expect this to cause problems on the cluster side when submitting the assembly jar of Toree. Has this changed?